i18n/fa_IR/code.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
Use this file to override default Docusaurus labels without swizzling the components